sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/MockSQLProjectDAO.java

```diff
@@ -16,7 +17,7 @@
  * @date 08/10/12
  * @since 0.1.9
  */
-public class MockSQLProjectDAO extends SQLProjectDAO {
+public class MockSQLProjectDAO extends HibernateProjectDao {
```

Comment:
Class not used anywhere. Likely replaced by Mockito :fire: 
MockSQLSecurityDAO is also unused while you're at it :fire: :fire: :fire: